File: articles/algorithms-in-decision-science/fortran/expected_value.f90
program expected_value
  implicit none
  real :: probability, benefit_if_act, cost_if_act, ev
  probability = 0.82
  benefit_if_act = 0.88
  cost_if_act = 0.30
  ev = probability * benefit_if_act - cost_if_act
  print *, "expected_value_of_action=", ev
end program expected_value

File: articles/algorithms-in-decision-science/rust/expected_value.rs
fn main() {
    let probability = 0.82_f64;
    let benefit_if_act = 0.88_f64;
    let cost_if_act = 0.30_f64;
    let expected_value = probability * benefit_if_act - cost_if_act;
    println!("expected_value_of_action={:.4}", expected_value);
}
